include/eos/fitting/ceres_nonlinear.hpp

```diff
@@ -30,29 +33,46 @@
 #include "glm/gtc/quaternion.hpp"
 #include "glm/gtx/transform.hpp"
 
+#include "ceres/ceres.h"
 #include "ceres/cubic_interpolation.h"
+#include "ceres/problem.h"
 
+
+#ifndef EOS_CERES_USE_OPENCV
```

Comment:
strange naming, why `CERES`?

Reply:
It prevent opencv usage only for ceres approach